### ISSUE TITLE
New release of Idris1 to hackage to build Idris2

### ISSUE THREAD

**Issue by shmish111** [edwinb/Idris2-boot#20]
Could someone release the latest idris package to hackage. I want a more stable build environment for idris 2 using nix but really to do that I need the version of idris 1 required to be on Hackage so that I can update nixpkgs etc etc.

**Comment by ysangkok** [comment]
I presume you also cannot build with the latest version, which is 1.3.1?

This is what I got:

```
Type checking ./Core/Context.idr
./Core/Context.idr:1815:7-1830:20:
     |
1815 |     = do opts <- getSession
     |       ~~~~~~~~~~~~~~~~~~~~~ ...
When checking right hand side of logTime with expected type
        Core a

When checking an application of function Core.Core.coreLift:
        No such variable clockTime

Makefile:12: recipe for target 'idris2' failed
make: *** [idris2] Error 1
$ idris --version
1.3.1
```

**Comment by shmish111** [comment]
Yeah exactly, it looks like `clockTime` was added after the release of 1.3.1

**Comment by edwinb** [comment]
As INSTALL.md says, you need the latest master. I've added a note to README.md which I hope is more prominent. I will do a release as soon as I can, but it does take time to make sure it's done right, and I could use a bit of a break for a few days!

**Comment by shmish111** [comment]
Thanks @edwinb a release of idris1 will mean I can work on tools to build idris2 much more easily. Oh and you deserve your break 😄

**Comment by edwinb** [comment]
I've just released 1.3.2
